docs/reference/5-command-line-tools/1-goliothctl/goliothctl_device_config.md

```diff
@@ -8,33 +8,21 @@ hide_title: true
 ---
 ## goliothctl device config
 
-Write a setting to the device.
 
-### Synopsis
-
-Use the `golioth device config` command to write a setting to the device.
-
-This command will use the `serial` port so it's required your device be connected to the USB before you run the command.
-
-```
-goliothctl device config [flags]
-```
-
-### Examples
 
 ```
-# Add a device with multiple hardware IDs
-> goliothctl device create --name foobar --hwId AA:AA:AA:AA --hwId BB:BB:BB:BB
```

Comment:
Why the entire block is being removed?

Reply:
@miguel91it the example being removed here is for the wrong command. These docs are for `goliothctl device config set` while the removed example is for `goliothctl device create`.